exposed-core/src/main/kotlin/org/jetbrains/exposed/sql/SQLExpressionBuilder.kt

```diff
@@ -352,12 +352,14 @@ interface ISqlExpressionBuilder {
 
     /** Returns the specified [value] as a query parameter of type [T]. */
     @Suppress("UNCHECKED_CAST")
+    @ExperimentalUnsignedTypes
```

Comment:
This will produce a lot of warns on the projects even if ULong wasn't used. Maybe temporary comment this line and one bellow?

Reply:
fixed in 26d7b29